tests/timeseries/_battery_pool/test_battery_pool.py

```diff
@@ -662,7 +666,8 @@ async def run_capacity_test(setup_args: SetupArgs) -> None:
     await streamer.stop_streaming(batteries_in_pool[0])
     await asyncio.sleep(MAX_BATTERY_DATA_AGE_SEC + 0.2)
     msg = await asyncio.wait_for(capacity_receiver.receive(), timeout=waiting_time_sec)
-    assert msg is None
+    if isinstance(msg, Sample):
+        assert msg.value is None
```

Comment:
I wonder if we can say: `assert isinstance(msg, Sample) and msg.value is None`, so we always have an assert.  Same for below.